loader/src/main/java/net/neoforged/neoforgespi/language/IModLanguageLoader.java

```diff
@@ -31,10 +37,23 @@
  */
 public interface IModLanguageLoader {
     /**
-     * {@return the name of this loader, used to decide what loader should load a mod}
+     * @return the name of this loader, used to decide what loader should load a mod
      */
     String name();
 
+    /**
+     * @return the version of this loader
+     */
+    default String version() {
```

Comment:
TBH, I'm inclined towards requiring locators to provide their version through whatever means they want.

Reply:
I made it non-default, but I don't like how currently the same version code is in three different places.